tests/topotests/lib/common_config.py

```diff
@@ -33,6 +33,7 @@
 import StringIO
 import os
 import sys
+import ipaddr
```

Comment:
nit: I'm no python expert, but I think we've been trying to move to `ipaddress` so we don't require the `ipaddr` package anymore.

https://github.com/FRRouting/frr/commit/8b547a6dab841c878bdd3c048e5d89cb96ed70f4